index.html

```diff
@@ -33,5 +42,139 @@ <h3>I'm a software Developer, </h3>
+          a standard dummy text.
+        </p>
+        <ul class="stack-holder">
+          <li><a href="#">CSS</a></li>
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+        <button class="see-project"><a href="#">See project</a></button>
+    </section>
+    <section class="section-three">
+      <div class="background-holder">
+        <h1>Profesional Art Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <ul class="profesional-holder">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button class="project-btn"><a href="#">See project</a></button>
+    </section>
+    <section class="section-three">
+      <div class="background-holder">
+        <h1>Profesional Art Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <ul class="profesional-holder">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button class="project-btn"><a href="#">See project</a></button>
+    </section>
+    <section class="section-three">
+      <div class="background-holder">
+        <h1>Profesional Art Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <ul class="profesional-holder">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button class="project-btn"><a href="#">See project</a></button>
+    </section>
+    <section class="section-three">
+      <div class="background-holder">
+        <h1>Profesional Art Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <ul class="profesional-holder">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button class="project-btn"><a href="#">See project</a></button>
+    </section>
+    <section class="section-four">
+      <div class="about-me-ctn">
+        <h1>About me</h1>
+        <p>
+          Hello I&apos;m a software developer! I can help you build a product , feature or website Look through some of my work and experience!
+          If you like what you see and have a project you need coded, don&apos;t hesitate to contact me.
+        </p>
+        <button><a href="#">Get my resume</a></button>
```

Comment:
- [ ] Great work on adding button styles! However, there are still some adjustments needed:

    - The button styles for the active and hover states are currently missing. To ensure a consistent and visually appealing user experience, please add appropriate styles for these states as specified in the design.

Keep up the good work, and don't forget to address these adjustments! 👍🌟